.gitignore

```diff
@@ -9,6 +9,7 @@
 .metadata/
 .project
 .settings/
+/journal/
```

Comment:
The reason for this change is that before it would ignore any journal/ directory, which was fine since the only journal/ was the one that was created for the local journal/ when running bin/tachyon-start. However, now we have journal/ directories for source code, so this change makes it so it only ignores the journal directory in the root.